### ISSUE TITLE
[Bug]: aliyun mo cannot startup after upgrade "panic: Duplicate Family Name LOG"

### ISSUE THREAD

**Issue by loveRhythm1990** [matrixorigin/matrixone#10716]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):nightly-d8e84ac9
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

upgrade from `nightly-8911e24d` to `nightly-d8e84ac9`

I have three logset nodes: `default-log-0` `default-log-1` `default-log-2`, following is their logs:

the first logset `default-log-0`:
```
lr90@mo pulumi % kubectl logs default-log-0 -n freetier-01
/mo-service -cfg /tmp/tmp.y0FVhkhHNv -debug-http=:6060
{"level":"INFO","time":"2023/07/17 07:37:03.790606 +0000","caller":"logutil/internal.go:37","msg":"MO logger init, level=, log file=, stackLevel=panic"}
{"level":"INFO","time":"2023/07/17 07:37:03.790817 +0000","caller":"fileservice/local_fs.go:133","msg":"fileservice: memory cache initialized","fs-name":"LOCAL","config":{"MemoryCapacity":536870912,"DiskPath":"/var/lib/matrixone/data/local-cache","DiskCapacity":8589934592,"DiskMinEvictInterval":"7m0s","DiskEvictTarget":0.8}}
{"level":"INFO","time":"2023/07/17 07:37:04.899968 +0000","caller":"fileservice/s3_fs.go:146","msg":"fileservice: memory cache initialized","fs-name":"SHARED","capacity":536870912}
{"level":"INFO","time":"2023/07/17 07:37:04.900031 +0000","caller":"fileservice/s3_fs.go:166","msg":"fileservice: disk cache initialized","fs-name":"SHARED","config":{"MemoryCapacity":536870912,"DiskPath":"/var/lib/matrixone/data/s3-cache","DiskCapacity":8589934592,"DiskMinEvictInterval":"7m0s","DiskEvictTarget":0.8}}
{"level":"INFO","time":"2023/07/17 07:37:05.180841 +0000","caller":"fileservice/s3_fs.go:146","msg":"fileservice: memory cache initialized","fs-name":"ETL","capacity":536870912}
{"level":"INFO","time":"2023/07/17 07:37:05.180898 +0000","caller":"fileservice/s3_fs.go:166","msg":"fileservice: disk cache initialized","fs-name":"ETL","config":{"MemoryCapacity":536870912,"DiskPath":"/var/lib/matrixone/data/etl-cache","DiskCapacity":8589934592,"DiskMinEvictInterval":"7m0s","DiskEvictTarget":0.8}}
{"level":"INFO","time":"2023/07/17 07:37:05.181013 +0000","caller":"fileservice/local_fs.go:133","msg":"fileservice: memory cache initialized","fs-name":"SHARED","config":{"MemoryCapacity":536870912,"DiskPath":null,"DiskCapacity":8589934592,"DiskMinEvictInterval":"7m0s","DiskEvictTarget":0.8}}
panic: Duplicate Family Name LOG 00000000-0000-0000-0000-000000000000 SHARED

goroutine 1 [running]:
github.com/matrixorigin/matrixone/pkg/util/metric/stats.(*Registry).Register(0xc000066740, {0xc0014f83c0, 0x2f}, {0xc001789920, 0x1, 0x0?})
	/go/src/github.com/matrixorigin/matrixone/pkg/util/metric/stats/registry.go:39 +0x1bc
github.com/matrixorigin/matrixone/pkg/util/metric/stats.Register(...)
	/go/src/github.com/matrixorigin/matrixone/pkg/util/metric/stats/registry.go:97
main.(*Config).createFileService(0xc001444000, {0x2eda248, 0xc00005e020}, {0x2a84f3a, 0x5}, 0xc000f61180, 0x4?, {0xc000dc2c60, 0x24})
	/go/src/github.com/matrixorigin/matrixone/cmd/mo-service/config.go:272 +0x93d
main.startService({0x2eda248, 0xc00005e020}, 0x296a6e0?, 0xc001444000?, 0xc00007a6f8?)
	/go/src/github.com/matrixorigin/matrixone/cmd/mo-service/main.go:140 +0xf9
main.main()
	/go/src/github.com/matrixorigin/matrixone/cmd/mo-service/main.go:97 +0x2b5
```

the second logset `default-log-1`:
```
lr90@mo pulumi % kubectl logs default-log-1 -n freetier-01
/mo-service -cfg /tmp/tmp.0u5svjip53 -debug-http=:6060
{"level":"INFO","time":"2023/07/17 07:35:10.282783 +0000","caller":"logutil/internal.go:37","msg":"MO logger init, level=, log file=, stackLevel=panic"}
{"level":"INFO","time":"2023/07/17 07:35:10.282966 +0000","caller":"fileservice/local_fs.go:133","msg":"fileservice: memory cache initialized","fs-name":"LOCAL","config":{"MemoryCapacity":536870912,"DiskPath":"/var/lib/matrixone/data/local-cache","DiskCapacity":8589934592,"DiskMinEvictInterval":"7m0s","DiskEvictTarget":0.8}}
{"level":"INFO","time":"2023/07/17 07:35:11.586471 +0000","caller":"fileservice/s3_fs.go:146","msg":"fileservice: memory cache initialized","fs-name":"SHARED","capacity":536870912}
{"level":"INFO","time":"2023/07/17 07:35:11.586544 +0000","caller":"fileservice/s3_fs.go:166","msg":"fileservice: disk cache initialized","fs-name":"SHARED","config":{"MemoryCapacity":536870912,"DiskPath":"/var/lib/matrixone/data/s3-cache","DiskCapacity":8589934592,"DiskMinEvictInterval":"7m0s","DiskEvictTarget":0.8}}
{"level":"INFO","time":"2023/07/17 07:35:11.846172 +0000","caller":"fileservice/s3_fs.go:146","msg":"fileservice: memory cache initialized","fs-name":"ETL","capacity":536870912}
{"level":"INFO","time":"2023/07/17 07:35:11.846219 +0000","caller":"fileservice/s3_fs.go:166","msg":"fileservice: disk cache initialized","fs-name":"ETL","config":{"MemoryCapacity":536870912,"DiskPath":"/var/lib/matrixone/data/etl-cache","DiskCapacity":8589934592,"DiskMinEvictInterval":"7m0s","DiskEvictTarget":0.8}}
{"level":"INFO","time":"2023/07/17 07:35:11.846315 +0000","caller":"fileservice/local_fs.go:133","msg":"fileservice: memory cache initialized","fs-name":"SHARED","config":{"MemoryCapacity":536870912,"DiskPath":null,"DiskCapacity":8589934592,"DiskMinEvictInterval":"7m0s","DiskEvictTarget":0.8}}
panic: Duplicate Family Name LOG 00000000-0000-0000-0000-000000000001 SHARED

goroutine 1 [running]:
github.com/matrixorigin/matrixone/pkg/util/metric/stats.(*Registry).Register(0xc000066740, {0xc00180fe00, 0x2f}, {0xc001887920, 0x1, 0x0?})
	/go/src/github.com/matrixorigin/matrixone/pkg/util/metric/stats/registry.go:39 +0x1bc
github.com/matrixorigin/matrixone/pkg/util/metric/stats.Register(...)
	/go/src/github.com/matrixorigin/matrixone/pkg/util/metric/stats/registry.go:97
main.(*Config).createFileService(0xc000e68000, {0x2eda248, 0xc00005e020}, {0x2a84f3a, 0x5}, 0xc000e02a80, 0x4?, {0xc000eac270, 0x24})
	/go/src/github.com/matrixorigin/matrixone/cmd/mo-service/config.go:272 +0x93d
main.startService({0x2eda248, 0xc00005e020}, 0x296a6e0?, 0xc000e68000?, 0xc00007a6f8?)
	/go/src/github.com/matrixorigin/matrixone/cmd/mo-service/main.go:140 +0xf9
main.main()
	/go/src/github.com/matrixorigin/matrixone/cmd/mo-service/main.go:97 +0x2b5
```

the third logset `default-log-2`:
```
/mo-service -cfg /tmp/tmp.KCi4u3M8Wo -debug-http=:6060
{"level":"INFO","time":"2023/07/17 07:35:10.217529 +0000","caller":"logutil/internal.go:37","msg":"MO logger init, level=, log file=, stackLevel=panic"}
{"level":"INFO","time":"2023/07/17 07:35:10.217733 +0000","caller":"fileservice/local_fs.go:133","msg":"fileservice: memory cache initialized","fs-name":"LOCAL","config":{"MemoryCapacity":536870912,"DiskPath":"/var/lib/matrixone/data/local-cache","DiskCapacity":8589934592,"DiskMinEvictInterval":"7m0s","DiskEvictTarget":0.8}}
{"level":"INFO","time":"2023/07/17 07:35:11.488377 +0000","caller":"fileservice/s3_fs.go:146","msg":"fileservice: memory cache initialized","fs-name":"SHARED","capacity":536870912}
{"level":"INFO","time":"2023/07/17 07:35:11.488440 +0000","caller":"fileservice/s3_fs.go:166","msg":"fileservice: disk cache initialized","fs-name":"SHARED","config":{"MemoryCapacity":536870912,"DiskPath":"/var/lib/matrixone/data/s3-cache","DiskCapacity":8589934592,"DiskMinEvictInterval":"7m0s","DiskEvictTarget":0.8}}
{"level":"INFO","time":"2023/07/17 07:35:11.742202 +0000","caller":"fileservice/s3_fs.go:146","msg":"fileservice: memory cache initialized","fs-name":"ETL","capacity":536870912}
{"level":"INFO","time":"2023/07/17 07:35:11.742259 +0000","caller":"fileservice/s3_fs.go:166","msg":"fileservice: disk cache initialized","fs-name":"ETL","config":{"MemoryCapacity":536870912,"DiskPath":"/var/lib/matrixone/data/etl-cache","DiskCapacity":8589934592,"DiskMinEvictInterval":"7m0s","DiskEvictTarget":0.8}}
{"level":"INFO","time":"2023/07/17 07:35:11.742354 +0000","caller":"fileservice/local_fs.go:133","msg":"fileservice: memory cache initialized","fs-name":"SHARED","config":{"MemoryCapacity":536870912,"DiskPath":null,"DiskCapacity":8589934592,"DiskMinEvictInterval":"7m0s","DiskEvictTarget":0.8}}
panic: Duplicate Family Name LOG 00000000-0000-0000-0000-000000000002 SHARED

goroutine 1 [running]:
github.com/matrixorigin/matrixone/pkg/util/metric/stats.(*Registry).Register(0xc000142700, {0xc0000594a0, 0x2f}, {0xc0004ed920, 0x1, 0x0?})
	/go/src/github.com/matrixorigin/matrixone/pkg/util/metric/stats/registry.go:39 +0x1bc
github.com/matrixorigin/matrixone/pkg/util/metric/stats.Register(...)
	/go/src/github.com/matrixorigin/matrixone/pkg/util/metric/stats/registry.go:97
main.(*Config).createFileService(0xc000dd0000, {0x2eda248, 0xc000120000}, {0x2a84f3a, 0x5}, 0xc0004a9180, 0x4?, {0xc0001452c0, 0x24})
	/go/src/github.com/matrixorigin/matrixone/cmd/mo-service/config.go:272 +0x93d
main.startService({0x2eda248, 0xc000120000}, 0x296a6e0?, 0xc000dd0000?, 0xc00007a6f8?)
	/go/src/github.com/matrixorigin/matrixone/cmd/mo-service/main.go:140 +0xf9
main.main()
	/go/src/github.com/matrixorigin/matrixone/cmd/mo-service/main.go:97 +0x2b5
```

### Expected Behavior

the logset cluster should startup successfully

**Comment by aunjgr** [comment]
@reusee fileservice belongs to you

**Comment by reusee** [comment]
same to https://github.com/matrixorigin/matrixone/issues/10706

**Comment by loveRhythm1990** [comment]
closed since this issue duplicate with https://github.com/matrixorigin/matrixone/issues/10706, and it is tested that this bug fixed on commitID ac56ec384ac6093e19be34d542f5ed20b76ebc1b
